server/repository/src/db_diesel/name_link_row.rs

```diff
@@ -34,6 +35,8 @@ allow_tables_to_appear_in_same_query!(name_link, item);
 allow_tables_to_appear_in_same_query!(name_link, location);
 allow_tables_to_appear_in_same_query!(name_link, barcode);
 allow_tables_to_appear_in_same_query!(name_link, program_enrolment);
+allow_tables_to_appear_in_same_query!(name_link, encounter);
+allow_tables_to_appear_in_same_query!(name_link, clinician_link);
```

Comment:
Logically I would place these where they are used, i.e. in the encounter files. The name_lin_row doesn't need to know about encounters and barcodes etc...

Reply:
[c9ed290](https://github.com/msupply-foundation/open-msupply/pull/2744/commits/c9ed2905c1f745463e279725642fcc75c3297dab)